modules/sno-adding-worker-nodes-to-sno-clusters-manually.adoc

```diff
@@ -164,6 +164,38 @@ where:
 $ coreos-installer install --ignition-url=<hosted_worker_ign_file> <hard_disk>
 ----
 
+.. To manually enable DHCP, apply the following `NMStateConfig` CR to the {sno} cluster:
++
+[source,yaml]
+----
+apiVersion: agent-install.openshift.io/v1
+kind: NMStateConfig
+metadata:
+  name: nmstateconfig-dhcp
+  namespace: example-sno
+  labels:
+    nmstate_config_cluster_name: <nmstate_config_cluster_label>
+spec:
+  config:
+    interfaces:
+      - name: eth0
+        type: ethernet
+        state: up
+        ipv4:
+          enabled: true
+          dhcp: true
+        ipv6:
+          enabled: false
+  interfaces:
+    - name: "eth0"
+      macAddress: "AA:BB:CC:DD:EE:11"
+----
++
+[IMPORTANT]
+====
+The `NMStateConfig` CR is required for successful deployments of worker nodes with static IPs and for adding a worker node with dynamic IP if the {sno} was deployed with static IP. The cluster network DHCP does not automatically set these network settings for the new worker node.
```

Comment:
```suggestion
The `NMStateConfig` CR is required for successful deployments of worker nodes with static IP addresses and for adding a worker node with a dynamic IP address if the {sno} was deployed with a static IP address. The cluster network DHCP does not automatically set these network settings for the new worker node.
```
See the SSG: https://redhat-documentation.github.io/supplementary-style-guide/#_i
See the ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#i